src/test/java/edu/iis/powp/gui/TestPlotSoftPatterns.java

```diff
@@ -31,6 +33,9 @@ private static void setupPresetTests(Context context) {
 	    SelectTestFigureOptionListener selectTestFigureOptionListener = new SelectTestFigureOptionListener();
 		
 		context.addTest("Figure Joe 1", selectTestFigureOptionListener);
+		context.addTest("Figure Joe 2", e -> {
+            FiguresJoe.figureScript2(Application.getComponent(DriverManager.class).getCurrentPlotter());
+        });
```

Comment:
It is good to keep a convention used in the original code (class instance instead of lambda)